inference-engine/src/mkldnn_plugin/nodes/mkldnn_experimental_detectron_generate_proposals_single_image_node.cpp

```diff
@@ -324,6 +320,12 @@ void MKLDNNExperimentalDetectronGenerateProposalsSingleImageNode::initSupportedP
                          impl_desc_type::ref_any);
 }
 
+void MKLDNNExperimentalDetectronGenerateProposalsSingleImageNode::createPrimitive() {
```

Comment:
override `needShapeInfer` to return false always since output shape depends on `post_nms_count` only

Reply:
done, thanks